main_page.html

```diff
@@ -5,47 +5,105 @@
   <head>
 	<title>Main Page</title>
 	<link rel="stylesheet" type="text/css" href="css/upls_styles.css">
-	<script src="js/main_menu.js"></script>
+	<script defer src="js/main_menu.js"></script>
+	<script defer src="js/main_page.js"></script>
```

Comment:
Don't forget to include [these files](https://github.com/upeipls/UPLS/blob/4f6ba2191d46111c29551f71b8f9f2aeb47a19e4/group_email.html#L39-L40) too.